crates/rolldown/tests/rolldown/topics/bundler_esm_cjs_tests/0/_config.json

```diff
@@ -6,5 +6,9 @@
         "import": "./entry.js"
       }
     ]
-  }
+  },
+  "configVariants": [
+    { "format": "cjs" }
+  ],
+  "expectExecuted": false
```

Comment:
Should be `true`. Would be fixed in later PRs.

Should allow `_test.mjs` to be ran under format cjs.

Reply:
https://github.com/rolldown/rolldown/pull/2788